{
  "api": "ml.get_model_snapshot_upgrade_stats",
  "file": "/test/platinum/ml/upgrade_job_snapshot.yml",
  "name": "Test existing but corrupt snapshot",
  "origin": "yaml",
  "request": {
    "args": {
      "job_id": "upgrade-model-snapshot",
      "snapshot_id": "1234567890"
    }
  },
  "response": {
    "headers": {
      "content-length": "259",
      "content-type": "application/json",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "error": {
        "reason": "no snapshot upgrade is running for snapshot_id [1234567890]",
        "root_cause": [
          {
            "reason": "no snapshot upgrade is running for snapshot_id [1234567890]",
            "type": "resource_not_found_exception"
          }
        ],
        "type": "resource_not_found_exception"
      },
      "status": 404
    },
    "statusCode": 404
  }
}